### ISSUE TITLE
react-native-svg crashes on Snack (SDK 49, 50)

### ISSUE THREAD

**Issue by bockoblur** [expo/expo#27171]
### Minimal reproducible example

https://snack.expo.dev/@bocko/6300a6?platform=android

### What platform(s) does this occur on?

Android, iOS

### Did you reproduce this issue in a development build?

No (tested in Expo Go)

### Summary

All of my snacks that use react-native-svg stopped working (similar error) on Expo Go when I upgraded to SDK 49 or 50.  This happens on Android device, as well as in iOS and Android emulators. Web works fine.
Can't find any suggestions or fixes anywhere. 
Even the basic example from Expo documentation on SVG shows the similar error, so I put that as a minimal reproducible example.

`View config getter callback for component `RNSVGPath` must be a function (received `undefined`)`

Looks like a Snack bug to me... 

### Environment

Expo Go

### Expo Doctor Diagnostics

—

**Comment by hadnet** [comment]
Same thing here.

**Comment by wodin** [comment]
I can confirm.

This example:
```javascript
import * as React from 'react';
import Svg, { Circle, Rect } from 'react-native-svg';

export default function SvgComponent(props) {
  return (
    <Svg height="50%" width="50%" viewBox="0 0 100 100" {...props}>
      <Circle cx="50" cy="50" r="45" stroke="blue" strokeWidth="2.5" fill="green" />
      <Rect x="15" y="15" width="70" height="70" stroke="red" strokeWidth="2" fill="yellow" />
    </Svg>
  );
}
```

Causes this error:
```
https://snack.expo.dev/dist/app.bundle.js (1:17307)View config getter callback for component `RNSVGRect` must be a function (received `undefined`). (Device)
```

**Comment by byCedric** [comment]
Hi all! Thanks for the report. The underlying issue has been fixed in expo/snack#559. You might need to create a new Snack though, e.g. https://snack.expo.dev/@bycedric/react-native-svg-example?platform=android.

**Comment by bockoblur** [comment]
Thanks for superfast solution.
I can confirm that it works with new snack on SDK50

Just FYI, on SDK49 now we have a new error:

````
Unable to resolve module 'module://react-native/Libraries/Utilities/codegenNativeComponent.js'
(Device)
  Evaluating module://react-native/Libraries/Utilities/codegenNativeComponent.js
  Evaluating module://react-native-svg.js
````
